theforeman.org/pipelines/vars/foreman/2.5.groovy

```diff
@@ -0,0 +1,40 @@
+def foreman_version = '2.5'
```

Comment:
I think we want to include stream and maybe Ubuntu 2004 here. Let's check with @evgeni to be sure though

Reply:
After looking at the forklift versions, I think that you're right. I went ahead and updated this to include those two, but if they are not needed happy to remove them too.

Reply:
As a general note, starting from nightly rather than 2.4 is always a good idea. The only difference then would be `s/nightly/2.5/` and drop `fc29` from the client distros.

Reply:
Ack, I'm gonna go ahead and do that